src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IArrayElementReferenceExpression.cs

```diff
@@ -949,5 +949,561 @@ public void F(string[] args)
 
             VerifyOperationTreeAndDiagnosticsForTest<ElementAccessExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void ArrayElementReference_NoControlFlow()
+        {
+            string source = @"
+class C
+{
+    void M(int[] a1, int[,] a2, int i1, int i2, int i3, int result1, int result2)
```

Comment:
>i2 [](start = 44, length = 2)

Consider changing one of the indices, such as `i2` here, to be a `byte` or some other type that involves a conversion to be used as argument to indexer?

Reply:
Sure, I will tweak one of the tests to verify this scenario.